### PR TITLE
16 configure state manager

### PR DESCRIPTION
**Pull request by JosMcG**
Did npm install recoil. Added RecoilRoot to main.jsx, as well as recoil imports. Created the State.jsx file.

### CODE REVIEW

File: src/State.jsx
Comment (rmcguinness) [inline src/State.jsx:1]: Can you add an example state in the State.jsx file? A simple string and/or number?

File: src/App.jsx [hunk 1]
```diff
@@ -1,33 +1,13 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import State from './State.jsx'
 
 function App() {
   const [count, setCount] = useState(0)
```
Comment (rmcguinness) [inline src/App.jsx:6]: This can also be deleted.

**Review by rmcguinness — CHANGES_REQUESTED** [review]

File: src/State.jsx
Comment (JosMcG) [reply src/State.jsx:1]: I added the example in State.jsx and included a useRecoilState example in App.jsx

File: src/App.jsx (hunk 1 above)
Comment (JosMcG) [reply src/App.jsx]: I deleted the reace useState and replaced with a recoil example.

**Review by rmcguinness — APPROVED** [review]
Awesome work LGTM!

**Comment by CBlack86** [comment]
I looked over your branch and it looks good to me

**Comment by heathjackson** [comment]
It all looks good to me as well

**Pull request merged**